dev/src/main/java/org/eclipse/codewind/intellij/ui/tasks/InstallCodewindTask.java

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.codewind.intellij.ui.tasks;
+
+import com.intellij.openapi.progress.ProgressIndicator;
+import org.eclipse.codewind.intellij.core.Logger;
+import org.eclipse.codewind.intellij.core.ProcessHelper;
+import org.eclipse.codewind.intellij.core.cli.InstallUtil;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+import static org.eclipse.codewind.intellij.ui.messages.CodewindUIBundle.message;
+
+public class InstallCodewindTask extends CodewindProcessTask {
+
+    public InstallCodewindTask(Runnable onSuccess) {
+        super(null, message("InstallCodewindJobLabel"), onSuccess);
+    }
+
+    @Override
+    protected ProcessHelper.ProcessResult runProcess(@NotNull ProgressIndicator indicator) throws Exception {
+        ProcessHelper.ProcessResult result = null;
+        try {
```

Comment:
Don't use a try-catch block here, any thrown exception will be handled in the `onThrowable()` method of the superclass (CodewindProcessTask), which will also put up an error dialog for the user.

Reply:
I changed this in all three of the new tasks.